pycheribuild/projects/cross/cheribsd.py

```diff
@@ -1230,6 +1233,14 @@ def __init__(self, config: CheriConfig):
                     self.extra_kernels_with_mfs.append("GFE")
             else:
                 self.fatal("Unsupported architecture for FPGA kernels")
+        if self.buildFettKernels:
+            if self.compiling_for_riscv(include_purecap=True):
+                if self.crosscompile_target.is_hybrid_or_purecap_cheri():
+                    self.extra_kernels.append("CHERI_FETT")
+                else:
+                    self.extra_kernels.append("FETT")
+            else:
+                self.fatal("Unsupported architecture for FETT kernels")
```

Comment:
Might be better to make this a warning instead of fatal so you can put it in the global cheribsd section in the JSON config. But I don't really mind either way.

Reply:
Alternatively it'd be nice if cheribsd-riscv64-purecap extended from cheribsd-riscv64 (and cheribsd-riscv64-nocheri would be where you put things specifically for _that_).... :D

Reply:
Yes that would be nice. I think it should be easy to do that once I finally fix way config options are inherited.

Reply:
Good call.  Fixed now.